src/articles/announcements/index.md

```diff
@@ -0,0 +1,57 @@
+---
+uid: announcements
+---
+
+# Announcements
+
+Announcements allow you to send a message to several channels at once.
+You can schedule announcements to go out at a specific time, or send them immediately.
+
+## Create an announcement
+
+To create an announcement, open a Direct Message with Abbot and just type what you want to announce!
+Abbot will respond and ask you what you want to do next:
+
+<img src="/public/images/articles/announcements/reply-options.png">
+
+Select "Create announcement" to open the "Create Announcement" dialog:
+
+<img src="/public/images/articles/announcements/create-announcement.png">
+
+* **Send as Abbot**: By default, Abbot "[Pseudo-impersonates](xref:integrations.slack.access#pseudo-impersonation)" your user and posts the announcement as you.
+    If you want the announcement to come from Abbot, select this option.
+    If you've customized your Abbot Bot, this will use the custom name and avatar you've configured.
+
+* **Message Delivery**: How do you want to choose the channels you'll send the message to?
+    * **All Shared Channels** will send to all Slack Connect Channels in your workspace that Abbot is a member of.
```

Comment:
This is approximately correct, but _technically_ this sends to all rooms with tracking enabled. That can include non-Shared channels and can exclude Shared channels if tracking has been disabled.

Reply:
Clarifying and linking to a short blurb on "Disabling Conversation Tracking"